server/rest/tale.py

```diff
@@ -397,6 +398,18 @@ def updateTaleAccess(self, tale, access, publicFlags, public):
         return self._model.setAccessList(
             tale, access, save=True, user=user, setPublic=public, publicFlags=publicFlags)
 
+    @access.user(scope=TokenScope.DATA_READ)
+    @autoDescribeRoute(
+        Description('Remove user from access control list for a tale.')
+        .modelParam('id', model='tale', plugin='wholetale', level=AccessType.READ)
+        .errorResponse('ID was invalid.')
+        .errorResponse('Access was denied for the tale.', 403)
+    )
+    def relinquishTaleAccess(self, tale):
```

Comment:
Would it make sense to allow the user to lower their access level instead of removing access entirely? I might no longer need ADMIN or WRITE privileges?  This is hypothetical at the moment, but for the discussed verification use case (author submits for review) wouldn't it be reasonable for them to lower their access to READ or None?